code-challenges.test.js

```diff
@@ -24,25 +29,90 @@
 
 // b) Create the function that makes the test pass.
 
+//create fib variable
+//for fib define num values 
+//for loop states that the function will act on numbers greater than 2, but less than or equal to the example inputs (6,10). Function will iterate through the given parameters
+//return array of 6 and 10 in fib.
+
+
+
+
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 10; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
+--------------------------------------
+var i;
+var fib = [];
+
+fib[0] = 0;
+fib[1] = 1;
+for (i = 2; i <= 6; i++) {
+  fib[i] = fib[i - 2] + fib[i - 1];
+  console.log(fib[i]);
+}
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 // Expected output: [-9, 7, 9, 199]
 
-const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+// const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+// // Expected output: [-823, 7, 23]
+
+describe("filteredNumbers", () => {
+    it("takes in an array and returns a new array of only odd numbers sorted from least to greatest.",
+  () => }
+  expect(filteredNumbers(fullArr1, fullArr2)).toEqual(filteredNumbers)
 
 
 // b) Create the function that makes the test pass.
 
+//Pseudocode
+    //make function oddsArray
+    //utilize .filter to iterate through original array and return odd values
+    //.sort to sort array from least to greatest
+    //utilize modulo and index to identify oddvalues ^
+    //return array of odd number least to greatest
+
+   const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+
+    const siftedArray = (array, dataType) => {
+       return array.filter(value => {
+           return typeof value === dataType
+       })
+    }
+//    console.log(sifter(fullArr1, "number")) 
+   // ouput 
+   const numbersArray = [4, 9, 0, 8, 7, 199, -9]
+
+   let lowestToHighest = (array.sort((a, b) => a -b)
+   //console.log(lowestToHighest()
+   )
+    needToRefactor = lowestToHighest.filter(value => {
+    return value % 2 !==0
+   })
+    //console.log(" need to refactor", needToRefactor)
+
+
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("accumulatingSum", () => {
```

Comment:
Please look over the jest section in the syllabus. It will give you a clear approach to this section.